[2025-04-30T04:19:50.159+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-30T04:19:50.210+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_csv_pipeline_v2.load_data manual__2025-04-30T04:19:26.779061+00:00 [queued]>
[2025-04-30T04:19:50.258+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_csv_pipeline_v2.load_data manual__2025-04-30T04:19:26.779061+00:00 [queued]>
[2025-04-30T04:19:50.269+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-04-30T04:19:50.304+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): load_data> on 2025-04-30 04:19:26.779061+00:00
[2025-04-30T04:19:50.319+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=90) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-04-30T04:19:50.322+0000] {standard_task_runner.py:72} INFO - Started process 92 to run task
[2025-04-30T04:19:50.323+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'etl_csv_pipeline_v2', 'load_data', 'manual__2025-04-30T04:19:26.779061+00:00', '--job-id', '113', '--raw', '--subdir', 'DAGS_FOLDER/etl_pipeline.py', '--cfg-path', '/tmp/tmpqget7x8v']
[2025-04-30T04:19:50.325+0000] {standard_task_runner.py:105} INFO - Job 113: Subtask load_data
[2025-04-30T04:19:50.415+0000] {task_command.py:467} INFO - Running <TaskInstance: etl_csv_pipeline_v2.load_data manual__2025-04-30T04:19:26.779061+00:00 [running]> on host 5d323234f92c
[2025-04-30T04:19:50.584+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_csv_pipeline_v2' AIRFLOW_CTX_TASK_ID='load_data' AIRFLOW_CTX_EXECUTION_DATE='2025-04-30T04:19:26.779061+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-30T04:19:26.779061+00:00'
[2025-04-30T04:19:50.586+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-04-30T04:19:50.587+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-04-30T04:19:50.588+0000] {logging_mixin.py:190} INFO - Current task name:load_data state:running start_date:2025-04-30 04:19:50.211527+00:00
[2025-04-30T04:19:50.589+0000] {logging_mixin.py:190} INFO - Dag name:etl_csv_pipeline_v2 and current dag run status:running
[2025-04-30T04:19:50.590+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-04-30T04:19:50.614+0000] {load.py:26} WARNING - ⚠️ El archivo de salida ya existía y fue sobrescrito: data/output/superstore_clean.csv
[2025-04-30T04:19:50.658+0000] {load.py:30} INFO - ✅ Archivo final guardado en data/output/superstore_clean.csv
[2025-04-30T04:19:50.659+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-04-30T04:19:50.675+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-04-30T04:19:50.678+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=etl_csv_pipeline_v2, task_id=load_data, run_id=manual__2025-04-30T04:19:26.779061+00:00, execution_date=20250430T041926, start_date=20250430T041950, end_date=20250430T041950
[2025-04-30T04:19:50.716+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-04-30T04:19:50.717+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-04-30T04:19:50.718+0000] {logging_mixin.py:190} INFO - Dag name:etl_csv_pipeline_v2 queued_at:2025-04-30 04:19:26.815817+00:00
[2025-04-30T04:19:50.719+0000] {logging_mixin.py:190} INFO - Task hostname:5d323234f92c operator:PythonOperator
[2025-04-30T04:19:50.781+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-04-30T04:19:50.811+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-04-30T04:19:50.814+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
